test/mitmproxy/utils/test_human.py

```diff
@@ -46,3 +46,8 @@ def test_pretty_duration():
     assert human.pretty_duration(10000) == "10000s"
     assert human.pretty_duration(1.123) == "1.12s"
     assert human.pretty_duration(0.123) == "123ms"
+
+
+def test_format_address():
+    assert human.format_address(('::1','55700','0','0')) == "[::1]:55700"
+    assert human.format_address(('::ffff:127.0.0.1','55700','0','0')) == "127.0.0.1:55700"
```

Comment:
Even if I use IPv4 address, the `client_conn.address` will convert it into IPv6 type.
eg. if I use `curl 127.0.0.1:8080`, client_conn.address[0] will be `::ffff:127.0.0.1`.
That's why I've used this assertion. 
Also, in the case of hostnames, client_conn.address will be an IPv6 address only, as you had pointed earlier in our slack discussion. This method takes a tuple having an IP address and identifies the version (v4 or v6).
So, aren't these tests enough for the `format_address` method?